front/src/modules/ui/layout/navbar/NavItem.tsx

```diff
@@ -69,7 +69,9 @@ const StyledSoonPill = styled.div`
   align-items: center;
   border-radius: 50px;
   background-color: ${(props) => props.theme.lightBackgroundTransparent};
-  font-size: ${(props) => props.theme.fontSizeExtraSmall};
+  font-size: ${(props) => props.theme.fontSizeExtraExtraSmall};
+  line-height: 7px;
```

Comment:
line-height: 7px looks to be a hack here. Maybe we should just enforce container to be 32px high + align-items: center no matter what line-height and font-size we have

Reply:
Yes sorry for that, when I change the font-size at first I was using the same `line-height` and I change it after, now the `SoonPill` has a fix height and proper alignment